modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -247,7 +246,8 @@ export default {
     async getMedia (options) {
       const qs = {
         ...this.filterValues,
-        page: this.currentPage
+        page: this.currentPage,
+        viewContext: this.relationshipField ? 'relationship' : 'manage'
```

Comment:
Same as piece manager. I don't know how this query builder works! But: it works!